provider/oauth2/tests.py

```diff
@@ -546,7 +536,7 @@ def test_password_grant_confidential(self):
         })
 
         self.assertEqual(200, response.status_code, response.content)
-        self.assertTrue(json.loads(response.content)['refresh_token'])
+        self.assertIn('refresh_token', json.loads(response.content))
```

Comment:
Please revert this.  The old way ensures that the refresh token is not falsey (`None`, etc).